### PR TITLE
Fixed multichannel vtk2sitk and added test

### PR DESCRIPTION
**Pull request by dave3d**

### CODE REVIEW

File: test/test_vtk.py [hunk 1]
```diff
@@ -27,3 +27,17 @@ def test_vtktositk():
     gc.collect()
 
     assert sitkimg[0, 0, 0] == 255.0
+
+def test_multichannel():
+    img = sitk.Image([10, 10], sitk.sitkVectorUInt8, 3)
+    img[0,0] = (255, 127, 42)
+    vtk_img = sitk2vtk(img)
+    print(vtk_img)
```
Comment (zivy) [inline test/test_vtk.py:35]: Please remove the print, doesn't help with understanding what went wrong when something isn't as expected.

**Review by zivy — CHANGES_REQUESTED** [review]

**Comment by blowekamp** [comment]
Thank you for working on this. What about sitk2vtk for multichannel images?

**Comment by dave3d** [comment]
> Thank you for working on this. What about sitk2vtk for multichannel images?

That works also. I guess I should explicitly test that too.

**Comment by blowekamp** [comment]
And please check if the documentation needs these details too.

**Review by zivy — APPROVED** [review]

**Pull request merged**